wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -37,6 +39,14 @@ class Robot : public frc::TimedRobot {
   static constexpr int kJoystickPort = 0;
 
   static constexpr double kElevatorKp = 5.0;
+  static constexpr double kElevatorKi = 0.0;
+  static constexpr double kElevatorKd = 0.0;
+
+  static constexpr units::volt_t kElevatorkS = 0.0_V;
+  static constexpr units::volt_t kElevatorkG = 0.0_V;
+  static constexpr auto kElevatorkV = 0.762_V * 1_s / 1_m;
+  static constexpr auto kElevatorkA = 0.762_V * 1_s * 1_s / (1_m);
```

Comment:
Redundant parentheses?

Reply:
You could write these as `0.762_V / 1_mps` and `0.762_V / 1_mps_sq` respectively to make the unit relationship clearer.